__tests__/app.test.js

```diff
@@ -3,6 +3,7 @@ const db = require("../db/connection");
 const testData = require("../db/data/test-data/index");
 const request = require("supertest");
 const app = require("../app");
+const articles = require("../db/data/test-data/articles");
```

Comment:
How come you've imported your test articles data here? This will already be present in testData